### PR TITLE
DAO-64 addressing comments 

### PR DESCRIPTION
**Pull request by Rekard0**
- Text Copy Update
- UI Bug Fixes

### CODE REVIEW

File: packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx [hunk 1]
```diff
@@ -45,9 +43,12 @@ const CreateDaoBasicInfo: React.FC<{
...
 
-  useEffect(() => {
-    setValue('isExistingToken', isExistingToken);
-  }, [isExistingToken, setValue]);
+  const methods = useForm<ICreateDaoBasicInfo>({ defaultValues: { ...basicInfo } });
+  const { control, watch, getValues, trigger } = methods;
+
+  // useEffect(() => {
+  //   setValue('isExistingToken', isExistingToken);
+  // }, [isExistingToken, setValue]);
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx]: if this useEffect() is no longer needed, can you remove it?

File: packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx [hunk 2]
```diff
@@ -67,6 +69,13 @@ const CreateDaoCollateral: React.FC<{
     formState: { errors },
   } = methods;
 
+  useEffect(() => {
+    if (isExistingToken) {
+      setValue('isScheduleNewDaoToken', 0);
+      setValue('isChallengeNewDaoToken', 0);
+    }
+  }, [isExistingToken, setValue]);
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx]: Can you explain why this useEffect is needed as it looks to me the form field's onChange() handler would update the values.

File: packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx (hunk 1 above)
Comment (Rekard0) [reply packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx]: done

File: packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx (hunk 2 above)
Comment (Rekard0) [reply packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx]: this use effect trigger once when component is mounted and if previouse basicInfo.isExistingToken is true.
this is needed, in the senarios of for example: isScheduleNewDaoToken set to true but user go back to basic info and change to using existing token, so when user reach this screen these variables are set to default again.

**Review by yuetloo — APPROVED** [review]

**Pull request merged**